gradle/libs.versions.toml

```
@@ -43,7 +43,7 @@ micronaut-build-plugins="7.6.4"
 micronaut-groovy = "4.7.0"
 micronaut-session = "4.6.0"
 micronaut-sql = "5.3.0"
-micronaut-test = "4.6.2"
+micronaut-test = "4.10.1"
```

```suggestion
micronaut-test = "4.10.2"
```

https://github.com/micronaut-projects/micronaut-test/releases/tag/v4.10.2